### DISCUSSION TITLE
Reorder parameters for Subclass in OpenAPI schema

### DISCUSSION THREAD

**Discussion by hashpipeline** [fastapi/fastapi#7674] [Questions] [answered]
### First check

* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation, with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.

### Description
How can I change the order of attributes in the OpenAPI docs so that first is the first parameter of the Request in the documentation?
```python
import uvicorn
from fastapi import FastAPI
from pydantic import BaseModel

app = FastAPI()

class Foo(BaseModel):
    second: str

class Bar(Foo):
    first: str

@app.post("/test")
def test(data: Bar):
    pass

if __name__ == '__main__':
    uvicorn.run(app, host='0.0.0.0', port=8010, reload=True)
```

### Additional context

Wrong order for my needs:
![wrong_order](https://user-images.githubusercontent.com/53627446/77101791-2920db80-6a18-11ea-9d4b-b9a62c64b821.PNG)

**Comment by tiangolo** [comment, marked as answer]
Have in mind that JSON objects keys are a data structure without key order.

But anyway, if you need them to show in a very specific order, you can declare your models exactly as you want them to show up instead of separating them with inheritance.

```Python
import uvicorn
from fastapi import FastAPI
from pydantic import BaseModel

app = FastAPI()


class Foo(BaseModel):
    second: str


class Bar(BaseModel):
    first: str
    second: str


@app.post("/test")
def test(data: Bar):
    pass

if __name__ == "__main__":
    uvicorn.run(app, host="0.0.0.0", port=8010, reload=True)
```

**Comment by hashpipeline** [comment]
Okay, thanks. That is at least a acceptable workaround even though it makes the data model elements less reuseable.
